openai/openai_response.py

```diff
@@ -9,6 +9,17 @@ def __init__(self, data, headers):
     @property
     def request_id(self) -> Optional[str]:
         return self._headers.get("request-id")
+    
+    @property
+    def retry_after(self) -> Optional[int]:
+        try:
+            return int(self._headers.get("retry-after"))
+        except ValueError:
```

Comment:
I think this will return a TypeError if the header isn't present

```
>>> int(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
```

Reply:
fixed